.github/workflows/ci.yml

```diff
@@ -66,15 +66,15 @@ jobs:
         ./linux-container-ci.sh ${{ env.BUILDER_VERSION }} aws-crt-${LINUX_BASE_IMAGE} build -p ${{ env.PACKAGE_NAME }} --compiler=${{ matrix.compiler }}
 
   windows:
-    runs-on: windows-latest
+    runs-on: windows-2019
```

Comment:
trivial: add comment to make it clear WHY we're using this particular version like
`runs-on: windows-2022 # latest`